build.gradle

```diff
@@ -18,6 +18,15 @@ allprojects {
   }
 }
 
+ext {
```

Comment:
It's nice @jonathan-caryl. But the build got failed. Please fix it before I can merge :)